test/jdk/java/awt/print/PageFormat/SetOrient.java

```diff
@@ -21,31 +21,83 @@
  * questions.
  */
 
-/**
- * @bug 4186119: setting orientation does not affect printer
+/*
+ * @test
+ * @key printer
+ * @bug 4186119
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
  * @summary Confirm that the clip and transform of the Graphics2D is
  *          affected by the landscape orientation of the PageFormat.
- * @run applet/manual=yesno SetOrient.html
+ * @run main/manual SetOrient
  */
 
-import java.awt.*;
-import java.awt.geom.*;
-import java.awt.print.*;
-import java.applet.Applet;
+import java.awt.Color;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.geom.Ellipse2D;
+import java.awt.print.Book;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
+
+public class SetOrient {
+    static PrinterJob pjob;
```

Comment:
Can be moved inside `init()` and even you change the method name since applet is removed.

Reply:
Renamed method to `createAndShowGUI`. Also moved `pjob` var into method.